.github/workflows/build.yml

```
@@ -0,0 +1,74 @@
+          key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-
+
+      - name: pnpm install
+        run: docker run --rm -v `pwd`:/home:rw --user "$(id -u):$(id -g)" mew-build-container /bin/bash -c "pnpm install --silent"
+
+      - name: build
+        env:
+          VIRUS_TOTAL_API_KEY: ${{secrets.VIRUS_TOTAL_API_KEY}}
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2005:style:9:23: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'

(shellcheck)